### ISSUE TITLE
List incomplete

### ISSUE THREAD

**Issue by 4everR4al** [poisk-ls/best-modded-site#7]
How this list of mod sites can complete if Platinmods is not listed? Nearly every page you listed steals the mods from there. It's basically the original source of the most mods you can find. As well they have testing team for shared mods to assure everything is clean even if they share it from other places and you can actually chat with the modders and administration which you cannot do on any of the sites you listed. Being in the modding scene for years and my no1 is Platinmods, tried nearly all pages you listed also and they cannot compete to be honest, beside the fact they copy from Platinmods anyways.

Please overwork the list, it is not accurate at all in the moment. But I like the project generally.

**Comment by poisk-ls** [comment]
well platinmods is "private modding forumn"

**Comment by 4everR4al** [comment]
Correct. But it is the source of mainly everything you listed and the source where they copy from. In my opinion, if you want to provide a useful list, then it needs to be complete. And you clearly left out the best out of them all. 

I like your project, that's why I want it to be better.

**Comment by poisk-ls** [comment]
as you said
